### PR TITLE
Events page

### PR DESCRIPTION
**Pull request by maxharrisnet**
Events
- Added new card styles for events and the cards with gradients

Global
- Moved hero styles to global, adjusted page markup accordingly
- Replacing H1 and adding mobile styles for fonts
- Shifted t-shirt sizing on some variables to add new ones, did find/replace to fix

### CODE REVIEW

File: css/global.css [hunk 1]
```diff
@@ -258,14 +258,18 @@ Variables (colors, text, spacing, etc.)
 	--font-weight-reg: 400;
 
 	/* Line Height */
-	--line-height-lg: 26px;
+	--line-height-xxl: 50px;
```
Comment (SolivanLau) [inline css/global.css:262]: Thank you for adding these!

File: css/global.css [hunk 2]
```diff
@@ -258,14 +258,18 @@ Variables (colors, text, spacing, etc.)
...
+	--line-height-lg: 32px;
+	--line-height-med: 26px;
 
 	/* Spacing */
 	--spacing-xxl: 140px;
 	--spacing-xl: 100px;
 	--spacing-lg: 80px;
 	--spacing-med: 40px;
 	--spacing-sm: 20px;
+	--spacing-xs: 10px;
```
Comment (SolivanLau) [inline css/global.css:273]: Just noticed a few needs for this - will update on my end as well!

File: css/global.css [hunk 3]
```diff
@@ -347,8 +353,8 @@ h5 {
 }
 
 p {
-	font-weight: 400;
-	line-height: 26px;
+	font-weight: var(--font-weight-reg);
```
Comment (SolivanLau) [inline css/global.css:357]: Thanks for replacing these with css vars!

File: css/global.css [hunk 4]
```diff
@@ -827,20 +848,47 @@ Media Queries (overriding styles for different screen sizes)
 	section {
 		padding: var(--spacing-med) 0;
 	}
+
+	.section-header {
```
Comment (SolivanLau) [inline css/global.css:854]: I am wondering if this can be combined with the rule above for section at line 851?

File: css/global.css [hunk 5]
```diff
@@ -868,7 +916,7 @@ Media Queries (overriding styles for different screen sizes)
 	===================================
 	*/
 
-	.hero h2 {
+	.hero h1 {
```
Comment (SolivanLau) [inline css/global.css:932]: Thank you for changing this!

File: js/cms.js [hunk 6]
```diff
@@ -0,0 +1,131 @@
+// Fetch page data from Strapi
```
Comment (SolivanLau) [inline js/cms.js:1]: Just wanted to confirm if we are using strapi or wordpress cms

File: css/events.css [hunk 7]
```diff
@@ -0,0 +1,232 @@
...
+	justify-content: center;
+	align-items: center;
+	gap: var(--spacing-sm);
+	padding: var(--spacing-med);
+}
+
+.events-container {
+	display: flex;
+	flex-flow: row wrap;
+	align-items: flex-start;
```
Comment (SolivanLau) [inline css/events.css:34]: Nice and DRY! I am noticing a few breakpoints could be added in to accommodate for the events wrapping in some strange ways. I think making some adjustments on flex layout or images for the following viewport widths might help:

- 1507px --> 1080px
image

- 1070px --> 1024px
image

- 821px - 768px things seem to be on the left side
image

File: events.html [hunk 8]
```diff
@@ -0,0 +1,236 @@
...
+
+  <!-- Styles -->
+  <link rel="stylesheet" href="./css/global.css" />
+  <link rel="stylesheet" href="./css/events.css" />
+</head>
+
+<body>
+  <!-- Header Navigation-->
+  <header>
+    <h1 class="sr-only">If These Lands Could Talk</h1>
```
Comment (SolivanLau) [inline events.html:23]: Will need to remove on my end!

**Review by SolivanLau — CHANGES_REQUESTED** [review]
Things are looking good - Thank you for adding the variables, I propose just a few gallery adjustment for some view ports!

File: js/cms.js (hunk 6 above)
Comment (maxharrisnet) [reply js/cms.js:1]: WordPress. I need to clean this file up a bunch.

File: css/events.css (hunk 7 above)
Comment (maxharrisnet) [reply css/events.css:34]: Thanks for checking these sizes, will fix!

Comment (maxharrisnet) [reply css/events.css:34]: Updated, please test again @SolivanLau

Comment (SolivanLau) [reply css/events.css:34]: I think this looks good! nothing is shifting to left or right sides from my end!

**Review by SolivanLau — APPROVED** [review]

**Pull request merged**